Apps/Sandcastle/gallery/3D Models Coloring.html

```diff
@@ -90,6 +90,7 @@
     selectionIndicator : false,
     shadows : true
 });
+viewer.clockViewModel.shouldAnimate = true;
```

Comment:
I would recommend changing all of these to simple `viewer.clock.shouldAnimate`, since it's clearer and doesn't require the timeline/animation widget to be present.

Reply:
Also, does this particular example need animation?  May be better to just not animate here (and choose a time that shows shadows so they always toggle, which right now only happens if the clock happens to be at a certain time).